packages/utils/src/useI18n.ts

```diff
@@ -0,0 +1,50 @@
+import { useEffect, useState } from 'react';
+import { dynamicActivateI18n } from './dynamicActivatei18n';
+
+function detectLocale() {
+  const supportedLocales = ['en', 'nb', 'fi'];
```

Comment:
I see we've defined those in `lingui.config.js`, too. Would it make sense to define it once and import it e.g. here?